WebGL.md

```diff
@@ -0,0 +1,74 @@
+# Implementing Immutable Unity SDK for WebGL
+
+This guide provides focused instructions for implementing the Unity-Immutable-SDK in WebGL projects.
+
+For general information on the SDK, please refer to
+the [Immutable Unity SDK documentation](https://docs.immutable.com/sdks/zkEVM/unity).
+
+Live example can be found at https://immutable.github.io/unity-immutable-sdk/sample/webgl
+
+## WebGL Template Setup
+
+WebGL template is a configuration setting that lets you control the appearance of the HTML page, so that you can: test, demonstrate, and preview your WebGL application in a browser.
+
+A WebGL Template is always needed for creating a WebGL application and it will always be stored within **Assets > WebGL Templates** to be used. You can refer to [Unity: WebGL Templates](https://docs.unity3d.com/Manual/webgl-templates.html) for more information.
+
+A Custom WebGL Template is required to implement the Immutable Unity SDK in WebGL projects. The easiest way to create a new custom WebGL template is to make a copy of the built-in Default or Minimal templates, which are stored in corresponding subfolders under <Unity Installation> > PlaybackEngines > WebGLSupport > BuildTools > WebGLTemplates.
+
+Every Unity Project includes these templates by default. Copy a template and place it in your own **Assets > WebGLTemplates** folder, and rename it to something meaningful so you can identify your template later.
+
+Once you have created your own template, copy the following files from Passport package into the **Assets > WebGLTemplates** folder:
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/unity-webview.js`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/callback.html`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/logout.html`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/index.html` > `Passport/index.html`
```

Comment:
For other targets like mobile or desktop, will these files be compiled into the game too? 

Should these files sit at the root level of the Unity SDK? Something like: `unity-immutable-sdk/WebGLTemplates`?

Reply:
> For other targets like mobile or desktop, will these files be compiled into the game too?
yes it will be, is it a problem?

> Should these files sit at the root level of the Unity SDK?
sure it makes sense, will update